src/pages/settings/Profile/TimezoneSelectPage.js

```diff
@@ -122,25 +112,18 @@ class TimezoneSelectPage extends Component {
     render() {
         return (
             <ScreenWrapper includeSafeAreaPaddingBottom={false}>
-                {({safeAreaPaddingBottomStyle}) => (
```

Comment:
We removed `safeAreaPaddingBottomStyle` why, won't it have any effect?

Reply:
Good catch! I removed in the screen and was supposed to have added inside the component. Fixed, thanks!